### PR TITLE
[BK-SUPPORT] Enable BookKeeper CI

### PR DESCRIPTION
**Pull request by zymap**
---

*Motivation*

Enable BookKeeper CI environment to run the tests.

*Modifications*

- Add Github Action to run BookKeeper tests

### CODE REVIEW

**Comment by zymap** [comment]
@sijie  PTAL. Thanks.

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+version: '3'
```
Comment (sijie) [inline docker/docker-compose.yml]: If this docker compose file is only used for bookkeeper testing, please move these files to `test/docker/bookkeeper`.

**Comment by zymap** [comment]
@sijie Moved. PTAL. Thanks.

File: .github/workflows/bookkeeper.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Run bookKeeper service
+      run: |
+        pushd test/docker
+        docker-compose pull
+        docker-compose up -d
+        popd
+        sleep 10
```
Comment (sijie) [inline .github/workflows/bookkeeper.yml]: using `sleep` is a very bad practice in testing. It is non-deterministic and causes flakiness.
what you should do is to check if the bookkeeper cluster is ready.

**Comment by zymap** [comment]
@sijie PTAL. Thanks.

**Pull request merged**